src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx

```diff
@@ -265,10 +268,20 @@ describe('App', () => {
         ),
       );
 
-      const statusFilters = document.querySelectorAll('va-checkbox');
-
-      // Only 3 unique statuses, so length should be 3
-      expect(statusFilters.length).to.eq(3);
+      const statusFilters = screen.getAllByTestId(/status-filter_/);
+      const filterNames = statusFilters.map(filter => filter.name);
+
+      const orderedStatuses = [
+        'On Hold',
+        'In Manual Review',
+        'Appealed',
+        'Claim Submitted',
+        'Closed',
+        'In Process',
+        'Incomplete',
+        'Saved',
+      ];
+      expect(filterNames).to.eql(orderedStatuses);
```

Comment:
Because set operation can be non intuitive, can we add tests for these cases?

- A user's claims have ONLY a subset of the "top statuses"
- A user's claims DO NOT have any of the "top statuses"
- A user's claims all have the SAME "non-top status"
- A user's claims all have the SAME "top status"